jkube-kit/doc/src/main/asciidoc/inc/build/_assembly.adoc

```diff
@@ -6,35 +6,41 @@ logic (see <<jkube-plugin, JKube Plugin>> for more details).
 
 [[config-image-build-assembly]]
 .Assembly Configuration (<<config-image, image >> : <<config-image-build, build >>)
-[cols="1,5"]
+[cols="1,5,1"]
 |===
-| Element | Description
+| Element | Description | Property
 
 | *name*
 | Assembly name, which is `maven` by default. This name is used for the archives and directories created during the build.
   This directory holds the files specified by the assembly. If an <<external-dockerfile,external Dockerfile>> is used then
   this name is also the relative directory which contains the assembly files.
+|
 
 | *targetDir*
 | Directory under which the files and artifacts contained in the assembly will be copied within the container.
   The default value for this is `${assembly.name}`, so `/maven` if *name* is not set to a different value.
+| `jkube.container-image.assembly.targetDir`
 
 | <<build-assembly-layer, *inline*>>
 | _Deprecated: Use layers instead_
   Inlined assembly descriptor as described in <<build-assembly-layer,Assembly - Inline>> below.
+|
 
 | <<build-assembly-layer, *layers*>>
 | Each of the layers that the assembly will contain as described in
   <<build-assembly-layer, Assembly - Layer>> below.
+|
 
 | *exportTargetDir*
 | Specification whether the `targetDir` should be exported as a volume. This value is `true` by default except in the
   case the `targetDir` is set to the container root (`/`). It is also `false` by default when a base image is used with
   `from` since exporting makes no sense in this case and will waste disk space unnecessarily.
+| `jkube.container-image.assembly.exportTargetDir`
 
 | *excludeFinalOutputArtifact*
```

Comment:
why is property not added for this field?

Reply:
It was not part of the original list of properties. We can certainly add it (Maybe as a follow up)